src/openfermion/transforms/__init__.py

```diff
@@ -34,3 +34,4 @@
 from ._jordan_wigner import jordan_wigner
 from ._reverse_jordan_wigner import reverse_jordan_wigner
 from ._verstraete_cirac import verstraete_cirac_2d_square
+from ._projection import project_onto_sector
```

Comment:
make imports alphabetically unless you run into a circular dependency.

Reply:
Thanks for finding that.